content/trento-docs-site/contribution-upstream/contributing_components_nav_guide.adoc

```diff
@@ -0,0 +1,353 @@
+= Component Documentation Navigation Guide
+
+
+== Overview
+
+The auto build script, link:{script_path}[*{script_name}*], automatically generates Antora navigation for upstream component documentation by scanning their *{scanned_upstream_directories}* directories and creating a structured navigation file.
```

Comment:
Using variables to keep the document maintainable in case paths change

Variables can be found in antora-playbook.yml. 
In the future they will be extracted from the docs itself so it is easier to read and review the file, but this comes in the future